src/views/LotteryV2/components/BuyTicketsModal/TicketInput.tsx

```diff
@@ -106,8 +114,9 @@ const TicketContaier: React.FC<{
   const onChangeHandler = (event: React.KeyboardEvent, digitId: number) => {
     const currentKey = parseInt(event.key, 10)
 
-    if (event.key === 'e' || event.key === 'E' || event.key === '.') {
+    if (['e', 'E', '.', ',', '-', 'Unidentified'].includes(event.key)) {
```

Comment:
"Unidentified" is the `even.key` value emitted by mobile keyboard when you press on `.`, `-` or `,`
Android doesn't have the way to hide those buttons and for some reason `.` and `-` slips through if you click multiple times

Reply:
ugh, actually just tested it on preview in TW - this doesn't solve the problem...